scripts/quests/sandoria/A_Job_for_the_Consortium.lua

```diff
@@ -0,0 +1,245 @@
+            {
+                onTrigger = function(player, npc)
+                    local damaged = quest:getVar(player, 'Prog') == packageCondition.DAMAGED and true or false
+
+                    if damaged then
+                        return quest:progressEvent(218)
+                    else
+                        return quest:progressEvent(219)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [218] = function(player, csid, option, npc)
+                    player:messageSpecial(lowerID.text.DAMANGED_PACKAGE_DELIVERED, xi.ki.BRUGAIRE_GOODS)
+                    player:delKeyItem(xi.ki.BRUGAIRE_GOODS)
+                end,
+
+                [219] = function(player, csid, option, npc)
+                    player:messageSpecial(lowerID.text.PACKAGE_DELIVERED, xi.ki.BRUGAIRE_GOODS)
+                    player:delKeyItem(xi.ki.BRUGAIRE_GOODS)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return vars.Stage == 1 and
+            not player:hasKeyItem(xi.ki.BRUGAIRE_GOODS)
+        end,
+
+        [xi.zone.PORT_SAN_DORIA] =
+        {
+            ['Portaure'] =
+            {
+                onTrigger = function(player, npc)
+                    local result = quest:getVar(player, 'Prog')
+                    -- 0 delivered, 1 = confiscated, 2 = damaged
+                    return quest:progressEvent(653 + result)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [653] = function(player, csid, option, npc)
+                    quest:complete(player)
+                    npcUtil.giveCurrency(player, 'gil', 1000)
+                    player:addFame(xi.fameArea.NORG, 30)
+                end,
+
+                [654] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+
+                [655] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+            },
+        },
+    },
+
+    -- Actions for Airship travels in Jeuno when suspended.
+    {
+        check = function(player, status, vars)
+            return player:getVar('[AIRSHIP]suspended') >= GetSystemTime()
+        end,
+
+        [xi.zone.PORT_JEUNO] =
+        {
+            ['_6u4'] = quest:event(42):setPriority(200), -- Sandy
+            ['_6u8'] = quest:event(41):setPriority(200), -- Kazahm
+            ['_6ua'] = quest:event(40):setPriority(200), -- Bastok
+            ['_6ue'] = quest:event(43):setPriority(200), -- Windurst
+            ['Illauvolahaut'] = quest:event(41):setPriority(200), -- Kazahm
+            ['Omiro-Zamiro']  = quest:event(43):setPriority(200), -- Windurst
+            ['Purequane']     = quest:event(42):setPriority(200), -- Sandy
+            ['Zedduva']       = quest:event(40):setPriority(200), -- Bastok
+        },
+    },
```

Comment:
From memory, this wont have the intended effect. This npcs have scripts which contain their logic. They will just cycle between the 2.
In other words, if IF stops you from taking an airship on first try, next trigger will just return their regular logic.

Reply:
I may be wrong though. But if I'm not, I would propose just adding this last piece of logic to their actual scripts.

Reply:
Default priority of scripts is 100, so setting any event over that will bypass the lua script.